packages/legend-studio/src/stores/LegendStudioRouter.ts

```diff
@@ -32,57 +30,23 @@ export enum LEGEND_STUDIO_PATH_PARAM_TOKEN {
   GAV = 'gav',
 }
 
-export const URL_PATH_PLACEHOLDER = '-';
-
-export const generateRoutePatternWithSDLCServerKey = (
-  pattern: string,
-): [string, string] => [
-  `/:${LEGEND_STUDIO_PATH_PARAM_TOKEN.SDLC_SERVER_KEY}(-)${pattern}`,
-  `/sdlc-:${LEGEND_STUDIO_PATH_PARAM_TOKEN.SDLC_SERVER_KEY}${pattern}`,
-];
-
 export const LEGEND_STUDIO_ROUTE_PATTERN = Object.freeze({
-  VIEW: generateRoutePatternWithSDLCServerKey(
-    `/view/:${LEGEND_STUDIO_PATH_PARAM_TOKEN.PROJECT_ID}`,
-  ),
-  VIEW_BY_GAV: `/view/:${LEGEND_STUDIO_PATH_PARAM_TOKEN.GAV}`,
-  VIEW_BY_GAV_ENTITY: `/view/:${LEGEND_STUDIO_PATH_PARAM_TOKEN.GAV}/entity/:${LEGEND_STUDIO_PATH_PARAM_TOKEN.ENTITY_PATH}`,
-  VIEW_BY_ENTITY: generateRoutePatternWithSDLCServerKey(
-    `/view/:${LEGEND_STUDIO_PATH_PARAM_TOKEN.PROJECT_ID}/entity/:${LEGEND_STUDIO_PATH_PARAM_TOKEN.ENTITY_PATH}`,
-  ),
-  VIEW_BY_REVISION: generateRoutePatternWithSDLCServerKey(
-    `/view/:${LEGEND_STUDIO_PATH_PARAM_TOKEN.PROJECT_ID}/revision/:${LEGEND_STUDIO_PATH_PARAM_TOKEN.REVISION_ID}`,
-  ),
-  VIEW_BY_VERSION: generateRoutePatternWithSDLCServerKey(
-    `/view/:${LEGEND_STUDIO_PATH_PARAM_TOKEN.PROJECT_ID}/version/:${LEGEND_STUDIO_PATH_PARAM_TOKEN.VERSION_ID}`,
-  ),
-  VIEW_BY_REVISION_ENTITY: generateRoutePatternWithSDLCServerKey(
-    `/view/:${LEGEND_STUDIO_PATH_PARAM_TOKEN.PROJECT_ID}/revision/:${LEGEND_STUDIO_PATH_PARAM_TOKEN.REVISION_ID}/entity/:${LEGEND_STUDIO_PATH_PARAM_TOKEN.ENTITY_PATH}`,
-  ),
-  VIEW_BY_VERSION_ENTITY: generateRoutePatternWithSDLCServerKey(
-    `/view/:${LEGEND_STUDIO_PATH_PARAM_TOKEN.PROJECT_ID}/version/:${LEGEND_STUDIO_PATH_PARAM_TOKEN.VERSION_ID}/entity/:${LEGEND_STUDIO_PATH_PARAM_TOKEN.ENTITY_PATH}`,
-  ),
-  REVIEW: generateRoutePatternWithSDLCServerKey(
-    `/review/:${LEGEND_STUDIO_PATH_PARAM_TOKEN.PROJECT_ID}/:${LEGEND_STUDIO_PATH_PARAM_TOKEN.REVIEW_ID}`,
-  ),
-  EDIT: generateRoutePatternWithSDLCServerKey(
-    `/edit/:${LEGEND_STUDIO_PATH_PARAM_TOKEN.PROJECT_ID}/:${LEGEND_STUDIO_PATH_PARAM_TOKEN.WORKSPACE_ID}/`,
-  ),
-  EDIT_GROUP: generateRoutePatternWithSDLCServerKey(
-    `/edit/:${LEGEND_STUDIO_PATH_PARAM_TOKEN.PROJECT_ID}/groupWorkspace/:${LEGEND_STUDIO_PATH_PARAM_TOKEN.GROUP_WORKSPACE_ID}/`,
-  ),
-  SETUP: generateRoutePatternWithSDLCServerKey(
-    `/setup/:${LEGEND_STUDIO_PATH_PARAM_TOKEN.PROJECT_ID}?/:${LEGEND_STUDIO_PATH_PARAM_TOKEN.WORKSPACE_ID}?`,
-  ),
-  SETUP_GROUP: generateRoutePatternWithSDLCServerKey(
-    `/setup/:${LEGEND_STUDIO_PATH_PARAM_TOKEN.PROJECT_ID}/groupWorkspace/:${LEGEND_STUDIO_PATH_PARAM_TOKEN.GROUP_WORKSPACE_ID}/`,
-  ),
+  NOT_FOUND: `/notfound`,
```

Comment:
As hopefully with this we stabalize our routes, might make sense to document this somewhere about what each url routes to.